examples/llava/qwen2vl-test.cpp

```diff
@@ -27,6 +27,8 @@
 #include <cassert>
 #include <cmath>
 
+// THIS FILE IS ONLY USED FOR TESTING THE QWEN2VL MODEL
+// IT IS NOT A PRODUCTION CODE
```

Comment:
@HimariO Please note that `llama-qwen2vl-cli` is no longer compiled, the target is removed from cmake. I still keep this file because it contains your code your testing M-RoPE